src/interfaces/searchOptions.type.ts

```diff
@@ -0,0 +1,5 @@
+export interface IOption {
```

Comment:
The name can be more descriptive I think. The name suggests I as abbreviation but not clear what.

Reply:
I changed it to `TaskSearchOption`. I couldn't come up with a better name than this.